pkg/genericcomp/fluentd_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package genericcomp
```

Comment:
The other functions in fluentd.go like `createFluentdVolHostPaths` are also accessible here - if it's possible to unit test some of those here it would be good to add the coverage

Reply:
fluentd.go has 100% coverage via genericcomponent_test.go and fluentd_test.go.  Is there a need to have  additional unit tests when there is already coverage?